src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
+  {
+    if (*val != 0)
```

Comment:
I can't really see any reason to move the parse part to Scope ...
What if in the future we want the q-string-filter in an operation that doesn't have Scope in the payload ?

About create/update subscriptions, I can't say I know.
Will look it up tomorrow.